spec/workers/content_changes_worker_spec.rb

```diff
@@ -0,0 +1,34 @@
+RSpec.describe ContentChangesWorker do
+  describe ".perform" do
+    shared_examples "tests for critical and warning states" do
+      context "find content changes from last 15 minutes" do
+        let(:statsd) { double }
+
+        before do
+          create(:content_change, created_at: 11.minutes_ago)
+          create(:content_change, created_at: 6.minutes_ago)
+          allow(GlobalMetricsService.send(:statsd)).to receive(:gauge)
+        end
+
+        it "records number of unprocessed content changes over 10 minutes old (critical)" do
+          expect(GlobalMetricsService).to receive(:critical_content_changes_total).with(1)
+          described_class.new.perform
```

Comment:
```suggestion
          subject.perform
```

I think by default `subject` is defined as `described_class.new`.

Reply:
This makes sense - however as `perform_async` is a class method, that test below uses `described_class`. To avoid mixing `subject` and `described_class` I might suggest leaving this in the more verbose form.

It also matches the style for the other `...Worker` specs which tend to use `described_class`